src/site/content/en/blog/largest-contentful-paint/index.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Largest Contentful Paint
```

Comment:
We've tried to avoid publishing articles that are just the name of an API because it's not immediately clear to the reader why they should read this post.

For example, https://web.dev/stale-while-revalidate/

Originally this post was just called "Stale while revalidate" but we changed it to "Keeping things fresh with stale-while-revalidate" and then updated the subhead to "An additional tool to help you balance immediacy and freshness when serving your web app."

Reply:
I'll update the subhead. For the title I think we need to keep it as is to fit with how we're structuring the future metrics section (e.g. one page per each metric).